webapp/src/main/java/com/box/l10n/mojito/service/security/user/UserService.java

```diff
@@ -385,4 +405,80 @@ public Page<User> findByUsernameOrName(String username, String search, Pageable
         });
     return users;
   }
+
+  /**
+   * Gets a service by name and if it doesn't exist create a created user.
+   *
+   * <p>All admin service accounts should already be created beforehand however, create a non-admin
+   * account to so long. If they need admin permissions, we will promote the account
```

Comment:
Think there's a typo here around `create a non-admin account to so long` but can't make out what it's supposed to mean

Reply:
Good spot. Thanks.
The comment was a tad outdated, so I've updated it